AdblockPlusSafariExtension/AdblockPlus+Parsing.m

```diff
@@ -0,0 +1,308 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-2015 Eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/&gt.
+ */
+
+#import "AdblockPlus+Parsing.h"
+
+// yajl is sax-like json parser. Content blocker extension has limited amount of memory,
+// so that it is not possible to load whole filter list at once.
+#include <yajl/yajl_parse.h>
```

Comment:
Is there a reason why the merging is not happening in the host app? Frankly, I thought that's kind of where  it belongs.